presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -128,6 +128,7 @@
     public static final String IGNORE_DOWNSTREAM_PREFERENCES = "ignore_downstream_preferences";
     public static final String REQUIRED_WORKERS_COUNT = "required_workers_count";
     public static final String REQUIRED_WORKERS_MAX_WAIT_TIME = "required_workers_max_wait_time";
+    public static final String SOURCE_TASK_COUNT_ESTIMATE = "source_task_count_estimate";
```

Comment:
you should also add corresponding feature config

Reply:
I was thinking of skipping feature config to avoid any modifications for all queries implicitly.